cli/tests/test_upgrade_public_resolver_bootstrap.py

```
@@ -0,0 +1,396 @@
+        )
+
+    assert raised.value.code == 0
+    assert run.call_args.args[0] == ["/bin/bash", "/private/resolver", "--yes"]
+
+
+@pytest.mark.parametrize(
+    "handoff",
+    (
+        {"DEFENSECLAW_UPGRADE_FRESH_PROCESS": "1"},
+        {"DEFENSECLAW_STAGED_UPGRADE": "1"},
+    ),
```

Ruff (0.15.21)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list`

Use `list` for parameter values

(PT007)